### PR TITLE
TC_08_04_01

### PR DESCRIPTION
**Pull request by MariaKM**
https://docs.google.com/document/d/1ff46TvvPYvH_AZfc4Ilqv6Dbo9Jxp31GQh-IJhrr0wc/edit?tab=t.0#heading=h.333obp2smjp7

https://app.qase.io/project/01?case=15&suite=11

### CODE REVIEW

**Review by SvetlanaMag — APPROVED** [review]

**Review by Volhabo — APPROVED** [review]
Ok

File: tests/08MyProfile.spec.js [hunk 1]
```diff
@@ -173,4 +181,61 @@ test.describe('My profile', () => {
...
+        await headerComponent.clickAccountSettingsLink();
+        await settingsGeneralPage.clickNotificationSettingsButton();
+
+        await step('Verify the "Manage your notifications" header is displayed.', async () => {
+            await expect(settingsNotificationsPage.notificationsHeading).toBeVisible();
+        });
+        const rows = await settingsNotificationsPage.notificationsTableRow;
+        const countRows = await rows.count();
+        await step('Verify the "Manage your notifications" table is displayed.', async () => {
+            await expect(settingsNotificationsPage.notificationsTableRow).toHaveCount(countRows);
```
Comment (annaafanaseva) [inline tests/08MyProfile.spec.js:214]: it seems you are retrieving countRows using rows.count(), but then comparing it back to the same settingsNotificationsPage.notificationsTableRow locator 🤔

File: tests/08MyProfile.spec.js [hunk 2]
```diff
@@ -173,4 +181,61 @@ test.describe('My profile', () => {
...
+        await step('Verify the "Email" notifications can be checked by default.', async () => {
+            expect(await settingsNotificationsPage.emailNotificationsCheckbox.all()).toBeChecked;
+        });
+
+        await step('Verify the "Browser" notifications can be checked by default.', async () => {
+            expect(await settingsNotificationsPage.browserNotificationsCheckbox.all()).toBeChecked;
+        });
+
+        await step('Verify the "Browser" notifications can be checked / unchecked.', async () => {
+            for (const checkbox of await settingsNotificationsPage.browserNotificationsCheckbox.all()) {
```
Comment (annaafanaseva) [inline tests/08MyProfile.spec.js:233]: I don't think this step is working properly...

toBeChecked is missing '()' - so it's not being executed
'await' is missing before check() and expect
to uncheck checkbox 'uncheck()' should be used

not sure you are 'checking' the correct element.. input reflects the checkbox state but click should happen on element below it - need to debug..

**Review by annaafanaseva — CHANGES_REQUESTED** [review]
please make necessary changes

**Review by annaafanaseva — APPROVED** [review]

**Review by annaafanaseva — APPROVED** [review]

**Pull request merged**